versions/2.0.0/asyncapi.md

```diff
@@ -1003,7 +1003,7 @@ Field Name | Type | Description
 <a name="messageObjectTags"></a>tags | [Tags Object](#tagsObject) | A list of tags for API documentation control. Tags can be used for logical grouping of messages.
 <a name="messageObjectExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this message.
 <a name="messageObjectBindings"></a>bindings | [Message Bindings Object](#messageBindingsObject) | A map where the keys describe the name of the protocol and the values describe protocol-specific definitions for the message.
-<a name="messageObjectExamples"></a>examples | [Map[`string`, `any`]] | An array with examples of valid message objects.
+<a name="messageObjectExamples"></a>examples | [Map[`string`, `any`]] | An array of key/value pairs where keys MUST be either "headers" and/or "payload". Values MUST contain examples of valid headers or payload definitions, respectively.
```

Comment:
`A map of two elements. The first one has a 'key'="headers" where a 'value' is an example of defined headers. The second element has a 'key'="payload" where a 'value' is an example of defined payload`.
...or something like this?...
...It will be great to set links to the correspond elements `headers` and `payload`...